internal/providers/providers.go

```diff
@@ -18,6 +25,32 @@ var prefixes = map[string]Provider{
 	"azapi_":      AZURE,
 }
 
+//go:embed azure_resource_tag_support.csv
```

Comment:
the csv is emebed to the executable.

Reply:
yeah, I think so. It's cleaner that way.
The CSV is embeded to the executable (nice golang feature). So from a user perspective there isn't a file.
The embed was done exactly for this use-case, to avoid having a large golang file holding an array. (better for compilation, code security, etc...).